app/Http/routes.php

```diff
@@ -57,6 +57,7 @@
 	Route::get('/profile', ['as' => 'member.profile', 'uses' => 'Profile\ProfileController@show'])->middleware('auth');
 	Route::post('/profile', ['as' => 'member.profile.update', 'uses' => 'Profile\ProfileController@save']);
 
+    Route::get('directory', ['as' => 'member.directory', 'uses' => 'Directory\DirectoryController@show'])->middleware('auth');
```

Comment:
oops, I didn't realise that the indentation of this line is actually alright: the problem was with lines 57-58. Hmm. Good to fix it though. You can either do it here in this review or in a separate one, up to you.

Incidentally, this is a good illustraton of the incentive for maintaining a clean diff - have a look at the git blame of routes.php - https://github.com/CUMSA/members/blame/bf74a52526f54d85006c45e7586b242703f359f5/app/Http/routes.php

If you have clean diffs and minimise unnecessary changes, you will be able to quickly pinpoint which commits/PRs introduced a line (which is mighty useful for debugging - often times you want to find the commits/PR to understand the line in the context of the other changes made, e.g. if you see a route and want to find out what functionality was added, this is a quick way of finding the relevant PR).